components/__init__.py

```diff
@@ -112,23 +112,6 @@ def define_mercury(reqs):
                           'make install'], libs=['opa'],
                 package='openpa-devel' if inst(reqs, 'openpa') else None)
 
-    if ARM_PLATFORM:
```

Comment:
is this no longer needed @vvenkates27  ?

Reply:
I'm including mchecksum on the DAOS side for a checksum timing util. I didn't think it was needed elsewhere, but I guess I'm not 100% sure. @vvenkates27 ??

Reply:
mchecksum is not needed for ARM anymore as ISA-L started supporting ARM. I actually had a patch locally to remove the requirement for using mchecksum in the checksum library for ARM . So I think we can ignore my patch #22, as this patch seems to do the same as well.